### DISCUSSION TITLE
how to install podman 4.0 on rhel8?

### DISCUSSION THREAD

**Discussion by rsarpal** [containers/podman#14135] [Q&A] [answered]
hi, 
I need to run k3d using podman and as per https://github.com/k3d-io/k3d/blob/main/docs/usage/advanced/podman.md k3d can be run with podman 4.0 onwards. 

dnf, yum and containertools doesnt have podman 4.0 build available. is there a binary / other way available to install podman 4.0 on RHEL 8?
br

**Comment by Luap99** [comment, marked as answer]
Podman 4.0 should land in RHEL 8.6 and RHEL 9.0
